zip-0208.rst

```diff
@@ -0,0 +1,131 @@
+
+Things that (might) have to change.
+
+Consensus protocol:
+ - difficulty adjustment constants after activation
+ - current values are
+   - PoWLimit := 2^243 - 1 for production
+                 2^251 - 1 for test
+   - PoWAveragingWindow := 17
+   - PoWMedianBlockSpan := 11
+   - PoWMaxAdjustDown := 32/100
+   - PoWMaxAdjustUp := 16/100
+   - PoWDampingFactor := 4
+   - PoWTargetSpacing := 150
+
+ - after Blossom activation
+   - PoWLimit is the same
+   - PoWAveragingWindow the same or increased? (probably same)
+   - PoWMedianBlockSpan the same or increased?
+   - PoWMaxAdjustDown/Up the same
+   - PoWDampingFactor the same
+   - PoWTargetSpacing changed
+
+ - emission per block after activation
+   - MaxBlockSubsidy := floor(1.25 * 10^9 / BlossomPoWTargetSpacingRatio) (zatoshis)
+
+ - halving block heights
+   - Halving(height) := { floor((height - SlowStartShift) / PreBlossomHalvingInterval), if height < BlossomActivationHeight
+                        { floor((BlossomActivationHeight - SlowStartShift) / PreBlossomHalvingInterval +
+                                (height - BlossomActivationHeight) / PostBlossomHalvingInterval), otherwise
+
+ - end of FR block height
+   - in this ZIP, don't change the FR consensus rule directly, but document the dependency on the split FR ZIP
+
+Other:
+ - default expiry blocks
+   - multiply by BlossomPoWTargetSpacingRatio if we want to maintain the same time
+ - EOS halt interval
+   - multiply by BlossomPoWTargetSpacingRatio if we want to maintain the same time
+
+ - sanity checks on nMinimumChainWork and fTransactionsPerDay don't need to change
+
+Maybe:
+ - difficulty target change at activation? not in this ZIP
+
+TODO: check for any other number-of-block constants.
+
+
+Deployment
+==========
+
+This proposal will be deployed with the Blossom network upgrade. [#zip-0206]_
+
+
+Backward compatibility
+======================
+
+This proposal intentionally creates what is known as a "bilateral consensus rule change". Use of this
+mechanism requires that all network participants upgrade their software to a compatible version within the
+upgrade window. Older software will treat post-upgrade blocks as invalid, and will follow any pre-upgrade
+branch that persists.
+
+
+Reference Implementation
+========================
+
+https://github.com/zcash/zcash/pull/xxxx
+
+
+References
+==========
+
+.. [#protocol] `Zcash Protocol Specification, Version 2018.0-beta-33 [Overwinter+Sapling] <https://github.com/zcash/zips/blob/master/protocol/protocol.pdf>`_
+.. [#RFC2119] `Key words for use in RFCs to Indicate Requirement Levels <https://tools.ietf.org/html/rfc2119>`_
+.. [#zip-0200] `ZIP 200: Network Upgrade Mechanism <https://github.com/zcash/zips/blob/master/zip-0200.rst>`_
+.. [#zip-0206] `ZIP 206: Deployment of the Blossom Network Upgrade <https://github.com/zcash/zips/blob/master/zip-0206.rst>`_
+.. [#zip-0207] `ZIP 207: Split Founders' Reward <https://github.com/zcash/zips/blob/master/zip-0207.rst>`_
```

Comment:
.. [#doublespend] `Table 1: The probability of a successful double spend <https://arxiv.org/pdf/1402.2009>`_

Reply:
.. [#krakensupport] `How long do digital assets/cryptocurrency deposits take? <https://support.kraken.com/hc/en-us/articles/203325283-How-long-do-digital-assets-cryptocurrency-deposits-take->`_

Reply:
.. [#slowfastblocks] `On Slow and Fast Block Times <https://blog.ethereum.org/2015/09/14/on-slow-and-fast-block-times/>`_